src/common/string_utils.cpp

```diff
@@ -0,0 +1,19 @@
+#include "string_utils.hpp"
+
+#include <algorithm>
+
+namespace rigel::strings {
+
+void split(std::vector<std::string> &output, std::string const &input,
+           char delimiter) {
+  auto start = std::begin(input);
```

Comment:
I wonder if we should `assert` that the delimiter is a plain ASCII character, i.e. less than 127? That way, string splitting can work correctly for UTF-8 encoded strings, without any surprises (as long as the delimiter is ASCII).

RigelEngine doesn't use Unicode anywhere so far, so it's not really a concern nowadays. But in the more distant future, I could imagine adding some localization features as an enhancement over the original game, so it would be a bit of future proofing.

What do you think?

Reply:
Will do.
Looks like something that can save some trouble in the long run, so I say why not.
Will be added along with the other changes.